sources/packages/api/src/database/entities/disbursement-schedule.model.ts

```diff
@@ -82,4 +82,34 @@ export class DisbursementSchedule extends RecordDataModel {
     },
   )
   disbursementValues?: DisbursementValue[];
+
+  /**
+   * COE approval status of disbursement.
+   */
+  @Column({
+    name: "coe_status",
+    type: "enum",
+  })
+  coeStatus: COEStatus;
+
+  /**
+   * User who approved/rejected COE.
+   */
+  @ManyToOne(() => User, { eager: false })
+  @JoinColumn({
+    name: "coe_updated_by",
+    referencedColumnName: ColumnNames.ID,
+  })
+  coeUpdatedBy: User;
+
+  /**
+   * Date on which COE is approved/rejected.
+   */
+  @Column({
+    name: "coe_updated_at",
+    type: "date",
+    transformer: dateOnlyTransformer,
```

Comment:
_dateOnlyTransformer_ is needed only for _date_ type. Please remove it.

Reply:
Done as changed type to timestamptz type